homeassistant/components/reolink/entity.py

```diff
@@ -11,12 +10,10 @@
 
 
 class ReolinkCoordinatorEntity(CoordinatorEntity):
-    """Parent class for Reolink Entities."""
+    """Parent class for Reolink camera entities."""
```

Comment:
The binary sensors also inherit this class. Please clarify what you mean.

Reply:
I changed it to "hardware camera entities".
I mean to make the diffrence between a camera device and a NVR (the two reolink product types).
A camera can be used stand allone (in this case each camera is a seperate integration instance).
Or multiple camera's can be connected to one NVR (in this case there is one integration instance controlling the NVR and all of its camera's with a device per camera).